googleAuth.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
```

Comment:
> Instead of adding an html file to the backend. Here's what I'd suggest. You can create a private repo on your gtihub acc then put this there then you'll add the url for that frontend to the cors of this repo so that it can have access and redirect @Aldot-02

Since the html file was for test purpose, allowing me to generate the token and testing if I can authenticate (both login and register) a new user, I have put it into a simulated react app so that I can use it in case of testing the functionality, or be used for front end integration in the future.